test/kwtool/crawlers_test.exs

```diff
@@ -0,0 +1,49 @@
+defmodule Kwtool.CrawlersTest do
+  use Kwtool.DataCase, async: true
+
+  alias Kwtool.Crawlers
+  alias Kwtool.Crawlers.Schemas.Keyword
+
+  describe "save_keywords_list/2" do
+    test "inserts the uploaded keywords to database" do
+      created_user = insert(:user)
+      keyword_file = %Plug.Upload{content_type: "text/csv", path: "test/fixture/keywords.csv"}
+
+      assert {:ok, "The keyword file is processed successfully!"} =
+               Crawlers.save_keywords_list(keyword_file, created_user)
+
+      [keyword1, keyword2, keyword3] = Repo.all(Keyword)
```

Comment:
How about having this an assertion too:

```
assert [keyword1, keyword2, keyword3] == Repo.all(Keyword)
```

Reply:
![Screen Shot 2021-04-27 at 3 23 21 PM](https://user-images.githubusercontent.com/948856/116209841-82575480-a76c-11eb-9388-77d2f7992e1c.png)

I updated it, to use the comparison `==` instead of binding `=`, however, I got the error like above.

Here, the idea is binding the return value from `Repo.all` to `[keyword1, keyword2, keyword3]`.
Not sure that is correct or not if we use the comparison 🤔